wildfly/connectors/file/connector-hdfs/src/main/java/org/teiid/resource/adapter/hdfs/HdfsManagedConnectionFactory.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags and
+ * the COPYRIGHT.txt file distributed with this work.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.teiid.resource.adapter.hdfs;
+
+import javax.resource.ResourceException;
+
+import org.apache.curator.shaded.com.google.common.base.Objects;
+import org.teiid.core.BundleUtil;
+import org.teiid.hdfs.HdfsConfiguration;
+import org.teiid.hdfs.HdfsConnection;
+import org.teiid.resource.spi.BasicConnectionFactory;
+import org.teiid.resource.spi.BasicManagedConnectionFactory;
+import org.teiid.resource.spi.ResourceConnection;
+import org.teiid.translator.TranslatorException;
+
+public class HdfsManagedConnectionFactory extends BasicManagedConnectionFactory implements HdfsConfiguration {
+
+    private static final long serialVersionUID = -687763504336137294L;
```

Comment:
What is this field?

Reply:
It was copied over from the ftpmanagedconnectionfactory.  Generally eclipse gives a warning is the serialVersionUID is not set on a serializable class.  We don't actually care about this concern, it's just to get rid of the warning.